doc/library/gof/wrapper.txt

```diff
@@ -0,0 +1,15 @@
+.. _libdoc_gof_wrapper:
+
+=============================================
+:mod:`wrapper` -- Wrapper class for op params
```

Comment:
Yes.  I know a lot of the doc doesn't do that yet, but it is a problem for linking inside the doc because those statements have meaning for sphinx.